dev-python/PythonQt/PythonQt-3.1.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit qmake-utils python-r1
+
+DESCRIPTION="A dynamic Python binding for the Qt framework"
+HOMEPAGE="http://pythonqt.sourceforge.net/"
+SRC_URI="mirror://sourceforge/pythonqt/pythonqt/${PN}-${PV}/${PN}${PV}.zip"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE="doc +extensions webkit"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	extensions? (
+		dev-qt/designer:5
+		dev-qt/qtdeclarative:5[widgets]
+		dev-qt/qtmultimedia:5[widgets]
+		dev-qt/qtnetwork:5
+		dev-qt/qtopengl:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtsql:5
+		dev-qt/qtsvg:5
+		dev-qt/qtxml:5
+		dev-qt/qtxmlpatterns:5
+	)
+	webkit? ( dev-qt/qtwebkit:5 )
+	"
+DEPEND="${RDEPEND}
+	dev-qt/qtxml:5
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )
+	"
+
+S="${WORKDIR}/${PN}${PV}"
+
+REQUIRED_USE="webkit? ( extensions ) ${PYTHON_REQUIRED_USE}"
+
+# Bring uitools back, backport other fixes from trunk
+PATCHES=( "${FILESDIR}/${P}-commit-433-441.patch" )
+
+src_prepare() {
+	default
+
+	if ! use extensions ; then
+		sed -i '/SUBDIRS/s/extensions//' ${PN}.pro || die "sed for extensions"
+	fi
+	if ! use webkit ; then
+		# Remove webkit support if not used
+		sed -i '/qtHaveModule(webkit):CONFIG += PythonQtWebKit/d' \
+			extensions/PythonQt_QtAll/PythonQt_QtAll.pro \
+			|| die "sed for webkit"
+	fi
+
+	# Unset python version to use python-config
+	sed -i "/unix:PYTHON_VERSION=/s/2.7//" build/python.prf \
+		|| die "sed for python version"
+}
+
+src_configure() {
+	python_foreach_impl eqmake5 PREFIX="${ED%/}"/usr
+}
+
+src_install() {
+	default
+
+	# Includes
+	insinto /usr/include/${PN}
+	doins -r src/*.h
+	insinto /usr/include/${PN}/gui
+	doins -r src/gui/*.h
+
+	if use extensions ; then
+		insinto /usr/include/${PN}/extensions/${PN}_QtAll
```

Comment:
Now I'm going to be picky but sometimes you are writing `PythonQt_QtAll` and sometimes `${PN}_QtAll`. Please settle on one. I personally prefer the former, I find constructing filenames out of ${PN} a bit confusing (i.e. requiring you to remember package name, not suitable for copy-paste) but that's your choice.